test/jdk/java/lang/reflect/AccessibleObject/TrySetAccessibleTest.java

```diff
@@ -193,11 +194,21 @@ public void testJavaLangClass() throws Exception {
 
         // non-public constructor
         Constructor<?> ctor
-            = Class.class.getDeclaredConstructor(ClassLoader.class, Class.class);
+            = Class.class.getDeclaredConstructor(ClassLoader.class, Class.class, ProtectionDomain.class);
         AccessibleObject[] ctors = { ctor };
 
         assertFalse(ctor.trySetAccessible());
         assertFalse(ctor.canAccess(null));
     }
 
+    /**
+     * Test that some Class fields cannot be made accessible.
+     */
+    public void testJavaLangClassFields() throws Exception {
+        try {
+            // This field is explicitly hidden from reflection.
+            Class.class.getDeclaredField("protectionDomain");
+            assertTrue(false);
+        } catch (NoSuchFieldException expected) { }
+    }
```

Comment:
The test is about accessibility, it's checking for IllegalAccessException and InaccessibleObjectException. So not the right place to test that a field is hidden from core reflection. Can you look at test/jdk/internal/reflect/Reflection/Filtering.java as that is probably the right place to list the protectionDomain field.

Reply:
Thank you Alan for letting me know the right place for this test.